doc/release_notes/release_notes_1.5.rst

```diff
@@ -0,0 +1,328 @@
+- :acrn-issue: '4228' -[acrn-configuration-tool] cannot boot hypervisor on customer board with KBL 7300U
+- :acrn-issue: '4229' -Add range check in Kconfig.
+- :acrn-issue: '4230' -Remove MAX_VCPUS_PER_VM in Kconfig
+- :acrn-issue: '4232' -Set default KATA_VM_NUM to 1 for SDC
+- :acrn-issue: '4247' -[acrn-configuration-tool] Generate Scenario for VM0 communites with VM1 is incorrect.
+- :acrn-issue: '4249' -[acrn-configuration-tool]Generated Launchscript but WebUI prompt error msg after we just select passthru-devices:audio_codec
+- :acrn-issue: '4255' -[acrn-configuration-tool][nuc7i7dnb][sdc]uos has no ip address
+- :acrn-issue: '4260' -[Community][External]webcam switch between 2 UOS.
+- :acrn-issue: '4286' -[acrn-configuration-tool] Remove VM1.vcpu_affinity.pcuid=3 for VM1 in sdc scenario
+
+Known Issues
+************
+TBD
+
+Change Log
+**********
+
+These commits have been added to the acrn-hypervisor repo since the v1.4
+release in Nov 2019 (click on the CommitID link to see details):
+
+.. comment
+
+   This list is obtained from this git command (update the date to pick up
+   changes since the last release):
+
+   git log --pretty=format:'- :acrn-commit:`%h` - %s' --after="2020-01-02"
+
+   - :acrn-commit:'ee74737f' - HV: search rsdp from e820 acpi reclaim region
+- :acrn-commit:'578a7ab4' - acrn-config: remove pcpu3 from vm1 in SDC scenario
+- :acrn-commit:'7d27c4bc' - hv: vpci: restore PCI BARs when doing AF FLR
+- :acrn-commit:'bb06f6f9' - hv: vpci: restore PCI BARs when doing PCIe FLR
+- :acrn-commit:'92ed8601' - hv: hotfix for xsave
+- :acrn-commit:'067d8536' - OVMF release v1.5
+- :acrn-commit:'9b71c5cd' - acrn-config: add 'logger_setting' into launch script
+- :acrn-commit:'be6c6851' - acrn-config: refine mount device for virtio-blk
+- :acrn-commit:'686d7763' - HV: Remove INIT signal notification related code
+- :acrn-commit:'d7eb14c5' - HV: Use NMI to replace INIT signal for lapic-pt VMs S5
+- :acrn-commit:'29b7aff5' - HV: Use NMI-window exiting to address req missing issue
+- :acrn-commit:'d26d8bec' - HV: Don't make NMI injection req when notifying vCPU
+- :acrn-commit:'24c2c0ec' - HV: Use NMI to kick lapic-pt vCPU's thread
+- :acrn-commit:'23422713' - acrn-config: add 'tap_' perfix for virtio-net
+- :acrn-commit:'6383394b' - acrn-config: enable log_setting in all vm
+- :acrn-commit:'0b44d64d' - acrn-config: check pass-thruogh device for audio/audio_codec
+- :acrn-commit:'75ca1694' - acrn-config: correct vuart1 setting in scenario config
+- :acrn-commit:'d52b45c1' - hv:fix crash issue when handling HC_NOTIFY_REQUEST_FINISH
+- :acrn-commit:'78139b95' - HV: kconfig: add range check for memory setting
+- :acrn-commit:'24994703' - HV: Kconfig: set default Kata num to 1 in SDC
+- :acrn-commit:'9d5e72e9' - hv: add lock for ept add/modify/del
+- :acrn-commit:'98b3dd94' - acrn-config: set HV_RAM_START above 256M for new board
+- :acrn-commit:'46463900000000' - acrn-config: add 'ramdisk_mod' item tag for tgl-rvp
```

Comment:
Does this commit number need to be fixed (or removed)? Sometimes these strange numbers mess up the Release Notes page.

Reply:
Done!